configuration/styles.css

```diff
@@ -6,6 +6,7 @@
 
 a {
     color: #fff;
+    cursor: pointer;
```

Comment:
The idea of the styles.css was to be able to easily swap out colors. Other css should not be located here .

Reply:
Solved